apiary-metastore-events/apiary-metastore-consumer/privileges-grantor/privileges-grantor-lambda/src/test/java/com/expediagroup/apiary/extensions/events/metastore/consumer/privilegesgrantor/lambda/PrivilegesGrantorTest.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.events.metastore.consumer.privilegesgrantor.lambda;
+
+
+import com.expediagroup.apiary.extensions.events.hive.consumer.common.exception.HiveClientException;
+import com.expediagroup.apiary.extensions.events.hive.consumer.common.thrift.ThriftHiveClient;
+import com.expediagroup.apiary.extensions.events.metastore.consumer.privilegesgrantor.core.Grantor;
+import com.expediagroup.apiary.extensions.events.metastore.consumer.privilegesgrantor.core.PrincipalName;
+import com.expediagroup.apiary.extensions.events.metastore.consumer.privilegesgrantor.core.Privilege;
+import com.expediagroup.apiary.extensions.events.metastore.consumer.privilegesgrantor.core.PrivilegesGrantor;
+import com.hotels.beeju.ThriftHiveMetaStoreJUnitRule;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.hadoop.hive.metastore.api.FieldSchema;
+import org.apache.hadoop.hive.metastore.api.HiveObjectPrivilege;
+import org.apache.hadoop.hive.metastore.api.HiveObjectRef;
+import org.apache.hadoop.hive.metastore.api.HiveObjectType;
+import org.apache.hadoop.hive.metastore.api.PrincipalType;
+import org.apache.hadoop.hive.metastore.api.SerDeInfo;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+import org.apache.hadoop.hive.metastore.api.Table;
+import org.apache.thrift.TException;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+public class PrivilegesGrantorTest {
+
+  private PrivilegesGrantor HiveClient;
```

Comment:
this variable doesn't look right ? It should be in lower case and should be named `privilegeGrantor` right ?

Reply:
yes, good catch!